api/src/test/java/org/openmrs/validator/AllergyValidatorTest.java

```diff
@@ -178,4 +178,18 @@ public void validate_shouldPassForAValidAllergy() {
 		
 		assertFalse(errors.hasErrors());
 	}
+	
+	@Test
+	public void valid_shouldRejectNumericAllergenValue() {
```

Comment:
You may also want to rename the method to reflect the method you're testing like

`public void validate_shouldRejectNumericAllergenValue() {}`

Reply:
Done the needful
Thanks @tendomart any other changes